clients/unshaded/src/main/java/tachyon/client/file/TachyonFileSystem.java

```diff
@@ -77,7 +77,7 @@ protected TachyonFileSystem() {
    * Convenience method for {@link #create(TachyonURI, CreateOptions)} with default options.
    */
   public TachyonFile create(TachyonURI path)
-      throws IOException, TachyonException, FileAlreadyExistsException, InvalidPathException {
```

Comment:
The subclasses of TachyonException are included to document the user-facing API (while the TachyonException is included so that we can introduce new specific exceptions in the future without breaking backwards compatibility of application that were written against the old API).

Unless you can suggest a different mechanism that provides for an informative and backwards compatible signature, I would prefer if we keep the "redundant" exceptions in the signature.